src/app/core/notifications/notification.model.js

```diff
@@ -2,13 +2,18 @@
 
 const mongoose = require('mongoose'),
 	deps = require('../../../dependencies'),
-	config = deps.config,
-	util = deps.utilService;
+	config = deps.config;
 
 /**
  * Notification Schema
  */
-module.exports.notificationOptions = { discriminatorKey: 'notificationType' };
+module.exports.notificationOptions = {
+	discriminatorKey: 'notificationType',
+	timestamps: {
+		createdAt: 'created',
+		updatedAt: 'updated'
+	}
```

Comment:
Schema didn't have an `updated` field since they are never updated.  The updated timestamp can be disabled by settings `updatedAt` to false.